frontend/src/app/Container.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import styles from './container.module.css'
+
+interface ContainerProps {
+    children: React.ReactNode
+    className?: string
+}
```

Comment:
Query: I'm not sure this className pattern will work so well with CSS Modules?

The way this is set up this would add a 'vanilla CSS' class name - doesn't this go against the fact we are using CSS Modules?

Also, in general, what extra classes would you likely want to pass to the Container component?

Reply:
Ah good point! 

As for the extra classes, colours, fonts are a couple examples but as this is css modules, you wouldn’t add it directly into the component anyway so I’ll remove it :)